### PR TITLE
EMP-292: Create CRM4 controller

### PR DESCRIPTION
**Pull request by muyinatech**
**Changes made:**
- Added types from crm4
- Added missing fields in crm5 types
- Created crm4 controller and tests
- Moved controllers to separate folder
- Added crm4 route

### CODE REVIEW

File: server/routes/index.test.ts [hunk 1]
```diff
@@ -39,124 +43,173 @@ afterEach(() => {
...
 
-describe('GET /', () => {
-  it('should render index page', () => {
-    return request(app)
-      .get('/')
-      .expect('Content-Type', /html/)
-      .expect(res => {
-        expect(res.text).toContain('Equiniti Historical Data')
-      })
+describe('routes', () => {
```
Comment (muyinatech) [inline server/routes/index.test.ts:46]: Just added a describe to surround all the tests, as it makes it easier to run all tests in the file in the IDE.

File: server/@types/crm4/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+type Quote = {
+  companyName: string
+  expertName: string
+  contactPhone: string
+  costBasis: string
+  preparationHours: number
+  hourlyRate: number
+  // typos to be fixed in BE code
```
Comment (sarahyjja) [inline server/@types/crm4/index.d.ts:89]: Do we need a ticket related to these typos?

Comment (muyinatech) [reply server/@types/crm4/index.d.ts:89]: In the past, a ticket has been raised on the BE first. Then once the changes are available they are made in the FE.
At the moment the typos are negligible, as the story for the display configuration has not been implemented yet.

Comment (muyinatech) [reply server/@types/crm4/index.d.ts:89]: Raising a ticket for the BE now.

**Pull request merged**